jenkins-integration/beaker/install/shared/install_deps.rb

```diff
@@ -0,0 +1,9 @@
+step "Configure epel" do
+  tmp_module_dir = master.tmpdir('configure_epel')
+  on(master, puppet('module', 'install', 'stahnma-epel', '--codedir', tmp_module_dir))
+  on(master, puppet('apply', '-e', "'include epel'"))
```

Comment:
I keep getting 

``` 
kmzh191du28xs3v.delivery.puppetlabs.net 18:35:09$ puppet apply -e 'include epel'
    Error: Evaluation Error: Error while evaluating a Function Call, Could not find class ::epel for kmzh191du28xs3v.delivery.puppetlabs.net  at line 1:1 on node kmzh191du28xs3v.delivery.puppetlabs.net
  
  kmzh191du28xs3v.delivery.puppetlabs.net executed in 2.10 seconds
  Exited: 1
  Beaker::Host::CommandFailure: Host 'kmzh191du28xs3v.delivery.puppetlabs.net' exited with 1 running:
   puppet apply -e 'include epel'
  Last 10 lines of output were:
  	[1;31mError: Evaluation Error: Error while evaluating a Function Call, Could not find class ::epel for kmzh191du28xs3v.delivery.puppetlabs.net  at line 1:1 on node kmzh191du28xs3v.delivery.puppetlabs.net
```

when running this.

It works if I do `puppet apply -e 'include epel' --codedir tmp_module_dir`.

Why are we using a tmp module dir though? And should we be cleaning it up? I've done 4 or 5 installs on this box now, and I've got a number of `tmp/configure_epel.<whatever>` directories now.